pennylane/resource/__init__.py

```diff
@@ -112,25 +125,34 @@ def circuit(theta):
 
 We can examine the resources by accessing the :code:`resources` key:
 
->>> resources_lst = tracker.history['resources']
->>> print(resources_lst[0])
-num_wires: 3
-num_gates: 7
-depth: 5
-shots: Shots(total=100)
-gate_types:
-{'RZ': 1, 'CNOT': 2, 'Hadamard': 2, 'PauliZ': 2}
-gate_sizes:
-{1: 5, 2: 2}
+.. code-block:: pycon
```

Comment:
Why do we need a `pycon` here ? If we want to silence doctest, that can also be done with `# doctest: +SKIP` instead.